script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -2792,13 +2797,13 @@ Not only that, but the sun feels strangely bright.
 -- Page 166, Offset 42467.
 -- 復元できているのは最低限の個所だけですから！　あと１時間はそのままでいてもらわないと！」
 // 'bare minimum amount of restoration' pretty clunky
-Your wounds have closed, but that's about it!
+Your wounds have only closed in a few vital spots, but that's about it!
```

Comment:
Doesn't really feel improved to me, would revert

Reply:
it's too simplified in the original
suggest alt

Reply:
I mean, the original's not all the descriptive. "Only the bare minimum has been restored!" and "Your wounds have closed, but that's about it!" seem like roughly equal semantic weight.
I guess I meant 'closed' to mean 'surface level, you're not _healed_', rather than 'closed and gone' maybe that's the disconnect? 

> You've not in critical condition anymore, but you're still [far from healthy/not out of the woods yet]!

Reply:
"Semantic weight" isn't the end-all-be-all. Ciel often talks in this more technical and precise manner.

As is, I don't really see the new suggestion as preferable

Reply:
I guess I'm fine with it if you at least drop the 'only', it conflicts with the 'but' as is